test/unittests/libtesteth/testHelper.cpp

```diff
@@ -0,0 +1,72 @@
+/*
+	This file is part of cpp-ethereum.
+
+	cpp-ethereum is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	cpp-ethereum is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file testHelper.cpp
+ * Unit tests for testHelper.
+ */
+
+#include <set>
+#include <string>
+#include <test/tools/libtesteth/TestOutputHelper.h>
+#include <test/tools/libtesteth/TestHelper.h>
+
+using namespace std;
+using namespace dev;
+using namespace dev::test;
+
+BOOST_FIXTURE_TEST_SUITE(TestHelperTestSuite, TestOutputHelperFixture)
+
+BOOST_AUTO_TEST_CASE(removeElementsFromSetTest_doubleElements)
+{
+	set<string> aSet;
+	aSet.insert("value1");
+	aSet.insert("value2");
+	aSet.insert("value1");
+	aSet = test::removeRepeatElementsFromSet(aSet);
+	BOOST_REQUIRE(aSet.size() == 2);
```

Comment:
Can you add another `BOOST_REQUIRE` before `test::removeRepeatElementsFromSet()`

Reply:
This test does not test anything.